scripts/bump.js

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+const { bold, underline } = require('chalk');
+const { spawnSync } = require('child_process');
+const inquirer = require('inquirer');
+const semver = require('semver');
+
+const REPO_DIR = path.dirname(__dirname);
```

Comment:
the double `dirname` here reads a little funny. maybe use `process.cwd()`, since this command will always be run from the platform root?

Reply:
It's actually intended so we can run it from any directory. Using `process.cwd()` makes an assumption where we run the script.

Reply:
that's true! do you anticipate running it without using `yarn bump` or from other directories often?

Reply:
Probably not often. Sometimes I create a symlink to common-used command and put it under `PATH` so I can run it fast from anywhere. `bump.js` isn't going to be a common command, but I think it's a good practice for scripts not to assume the current working directory.